### PR TITLE
Add / update MFDATA doc

### PR DESCRIPTION
**Pull request by dearith**
Add / update MFDATA doc (to be continued)

### CODE REVIEW

File: doc/mfdata_install.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# Installation
```
Comment (thefab) [inline doc/mfdata_install.md]: I think we can drop this file and keep the current installation guide

maybe we can copy it here (to be sure it is visible in the sphinx doc) in Makefile process ?

File: doc/plugins_guide.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# Plugins guide
```
Comment (thefab) [inline doc/plugins_guide.md]: See installation guide, same idea here

File: doc/mfdata_install.md (hunk 1 above)
Comment (dearith) [reply doc/mfdata_install.md]: Yes, we have to discuss how and where to make the copy (currently, the corresponding .md files are in the resources repo and not in the mfdata repo)

File: doc/plugins_guide.md (hunk 2 above)
Comment (dearith) [reply doc/plugins_guide.md]: Same as installation guide

File: doc/mfdata_install.md (hunk 1 above)
Comment (thefab) [reply doc/mfdata_install.md]: the version in "resources" is only for backward compatibility

the current version is available in ".metwork-framework" subdir of mfdata

**Pull request merged**